db/seeds.rb

```diff
@@ -2,14 +2,26 @@
 # The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
 #
 # Examples:
-admin= User
-       .where(email: "admin@fintech.com")
-       .first_or_create(firstname: "Muhammad", lastname: "Haroon",
-        password: "Password123!@#", user_role: User::ROLES[:admin])
+admin = User
+          .where(email: "admin@fintech.com")
+          .first_or_create(firstname: "Muhammad", lastname: "Haroon",
+            password: "Password123!@#", role: User::ROLES[:admin])
```

Comment:
This can also go up with `first_or_create`.

Reply:
Also, make an array of hashes of these attributes and create records in each loop as only values are different but code is duplicated. So we should DRY this.